source/telemetry.js

```diff
@@ -5,11 +5,15 @@ const version_helper = require('./version_helper.js')
 //// Log telemetry for Sequelize ORM.
 Sequelize.addHook('afterInit', async (connection) => {
     try {
-        var telemetryDisabled = connection.options.dialectOptions.cockroachdbTelemetryDisabled
-        if (telemetryDisabled) {
-            return 
+        var telemetryDisabled = false
```

Comment:
seems like we don't need this local var now, since it's only used/set in lines 10-11. (or just declare it on line 10 instead of here)

Reply:
Fixed, thanks!